Makefile

```diff
@@ -172,11 +172,11 @@ go.sum: go.mod
 update-swagger-docs: statik
 	$(BINDIR)/statik -src=client/docs/swagger-ui -dest=client/docs -f -m
 	@if [ -n "$(git status --porcelain)" ]; then \
-        echo "\033[91mSwagger docs are out of sync!!!\033[0m";\
-        exit 1;\
-    else \
-    	echo "\033[92mSwagger docs are in sync\033[0m";\
-    fi
+	      echo "\033[91mSwagger docs are out of sync!!!\033[0m";\
+	      exit 1;\
+	 else \
+	      echo "\033[92mSwagger docs are in sync\033[0m";\
+	 fi
```

Comment:
What's the rationale for this change? Please remove this to reduce diff size.

Reply:
This cmd could be removed I believe

Reply:
Yeah it can be.

Reply:
The spaces are not same in this task definition and my editor is saying: 
```
Suspicious line 178. Save anyway? (y or n) y
```
It's awkward when I need to do it every time I save the Makefile.
Clearing the formatting solves the issue.

Reply:
Just set a different option for Makefile in your editor. If we wanted to accommodate everyone's editor configuration we'd need to hire somebody full time for such a job. Please remove this patch's chunk, thanks!

Reply:
@alessio  - it's not about accommodating everyone's editor configuration. I'm not adding any configuration - making a style improvement. Why arguing if someone is doing small cleaning in the meantime, this is a positive change, right?

Anyway, if we can remove this command then even better.  I will remove it.

Reply:
I removed this code :)